frontend/src/metabase/visualizations/lib/settings.js

```diff
@@ -24,6 +25,7 @@ import ChartSettingColorsPicker from "metabase/visualizations/components/setting
 
 const WIDGETS = {
     input: ChartSettingInput,
+    inputs: ChartSettingInputs,
```

Comment:
I'm wondering if tweaking the naming here could help make things just a little more clear. `inputGroup: ChartSettingsInputGroup` or something? The closeness of the singular and plural here tingles my `future confusion` spidey sense. Comments explaining either way would be helpful.

Reply:
I'll change it but we still have other potentially confusing name pairs:  `FieldPicker`/`FieldsPicker` and `ColorPicker`/`ColorsPicker`. 

Sort of surprised they didn't go with `ColorPicker` and `ColourPicker` instead 😳